src/MakeHome.ZC

```diff
@@ -8,6 +8,7 @@ Cd(__DIR__);;
 #include "~/HomeWrappers"
 MapFileLoad("::/Kernel/Kernel");
 MapFileLoad("::/Compiler/Compiler");
+#include "::/Demo/Graphics/WallPaperDolDoc"
```

Comment:
@GutPuncher 
This won't display the wallpaper on boot, but it will allow for the HomeKeyPlugIns Alt+F10 / Alt+F11 to work.
I'd actually prefer not doing it this way.